text/0000-localizing-api-messages.md

```diff
@@ -0,0 +1,164 @@
+
+* Start Date: 2022-03-29
+* RFC PR: 
+  * Preliminary Review: https://github.com/folio-org/rfcs/pull/3
+  * Draft Review :
+  * Public Review :
+  * Final Review :
+* FOLIO Issue:
+* Current Status: :no_entry_sign: DRAFT REVIEW :no_entry_sign:
+* Submitter : Radhakrishnan Gopalakrishnan (rgopalakrishnan@ebsco.com)
+* Co-Submitter(s) : Zak Burke (zburke@ebsco.com)
+* Sub Group : 
+  * Marc Johnson (marc.johnson@k-int.com)
+  * Julian Ladisch (julian.ladisch@gbv.de)
+  * Peter Murray (peter@indexdata.com)
+  * Tod Olson (tod@uchicago.edu)
+
+# Localizing API (Backend) Messages
+
+## Summary
+  The purpose of this RFC is to define an approach that allows API developer to provide
+messages in the API response depending on the language preference set in the request. 
+
+
+## Motivation
+
+- Help users understand the information coming from the application backend using a language they understand better
+- To allow FOLIO platform to be used by users across the world 
+- To create a tremendous opportunity for growth that we can never could have achieved within just one country.
+- To Promote consistency and make it easier for tools and translators when handling translation files
+
+### Terminology
+* Base Language - Language on which all other translations are based.
+* Message Key - Refers to the string that is used in place of a hardcoded message in the code
+* Message Value - Refers to the string that is fetched using a message key lookup for a given locale
+* Translation file - Contains Message key and values for a given locale
+* Static Messages - Refers to the messages values stored in the translation file
```

Comment:
This definition seems to be dependent upon the way the messages are stored rather than why they are static (and different to dynamic)

```suggestion
* Static (design / compile time) Messages - Message values that are defined during development and do not vary by deployment environment
```